internal/resources/managementcluster/management_cluster_registration_test.go

```diff
@@ -31,12 +33,40 @@ func TestAcceptanceForManagementClusterRegistrationResource(t *testing.T) {
 	tKGsResourceName := fmt.Sprintf("%s.%s", "tanzu-mission-control_management_cluster", "test_tkgs")
 	tKGmResourceName := fmt.Sprintf("%s.%s", "tanzu-mission-control_management_cluster", "test_tkgm")
 
-	tkgsSimpleName := acctest.RandomWithPrefix("a-tf-tkgs-simple-test")
-	tkgmSimpleName := acctest.RandomWithPrefix("a-tf-tkgm-simple-test")
+	tkgsSimpleName := acctest.RandomWithPrefix("a-tf-tkgs-test")
+	tkgmSimpleName := acctest.RandomWithPrefix("a-tf-tkgm-test")
 
 	tkgmKubeconfigFilePathName := acctest.RandomWithPrefix("a-tf-tkgm-kubeconfig-filepath-test")
 
 	kubeconfigPath := os.Getenv("KUBECONFIG")
+	_, enablePolicyEnvTest := os.LookupEnv("ENABLE_POLICY_ENV_TEST")
```

Comment:
POLICY_ENV_TEST?

Reply:
I renamed it to **ENABLE_MANAGEMENT_CLUSTER_ENV_TEST** to have same approach as other mock tests flags.